gemfiles/Gemfile_rack_1

```diff
@@ -5,5 +5,6 @@ gemspec path: '..'
 gem 'byebug'
 gem 'wwtd'
 
-gem 'rack', '< 2'
-gem 'activesupport', '< 5'
+gem 'rack', '~> 1.0'
+gem 'activesupport', '~> 4.0'
+gem 'json', git: 'https://github.com/flori/json.git', tag: 'v1.8.3'
```

Comment:
this is not great. 2.3.1 drops the json pure parser in the json gem it includes. json pure parser is still in the json gem in rubygems, but I don't see a way to specify that it should use the rubygem instead of the included gem. I'm pointing at github for now, which is not the best (projects that use this will have to point to github for this to work too). I need to either figure a way to make it use json from rubygems, or just include the json pure parser in api hammer.